src/ixdat/tools.py

```diff
@@ -71,3 +80,159 @@ def list_is_close(list_one, list_two):
             return False
 
     return True
+
+
+def _construct_deprecation_message(
+    identity,
+    last_supported_release,
+    update_message,
+    hard_deprecation_release,
+    remove_release,
+    kwarg_name,
+):
+    """Return a deprecation message
+
+    Args:
+        identity (str): The identity of the callable being deprecated e.g.
+            "function 'myfunction'"
+
+    All other arguments are as in :func:`deprecate`
+
+    """
+    property_part = f"argument named '{kwarg_name}' in " if kwarg_name else ""
```

Comment:
Why not just make the `property_part` part of `identity` in the outer scope?

Reply:
Or maybe it is even cleaner to move it the other way. I will figure it out.